tools/add_triage_signature.py

```diff
@@ -839,6 +839,56 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket(report)
 
 
+class DownloadImage(ErrorSignature):
+    image_download_regex = re.compile(r"Host (.+?): New image status (.+?). result:.+?; download rate: (.+?) MBps")
+    minimum_download_rate_mb = 10
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="slow_image_download",
+            comment_identifying_string="h1. Host slow download rate:",
+        )
+
+    @classmethod
+    def _list_image_download_info(cls, events):
+        def get_image_download_info(event):
+            matches = cls.image_download_regex.findall(event["message"])
+            if len(matches) == 0:
+                return None
+            match = matches[0]
+            if len(match) < 3:
+                return None
+
+            return OrderedDict(id=event["host_id"], hostname=match[0], image=match[1], download_rate=match[2])
```

Comment:
python's dictionaries are already ordered since Python 3.7
```suggestion
            return dict(id=event["host_id"], hostname=match[0], image=match[1], download_rate=match[2])
```